src/test/create-user.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+import chai from 'chai';
+import { UserInput } from '../resolvers';
+import { dataSource } from '../data-source';
+import { User } from '../user';
+
+describe('Testing createUser Mutation', () => {
+  it('should return created user', async () => {
+    const user: UserInput = {
+      name: 'Taq',
+      email: 'taq@gmail.com',
+      password: 'forte123123',
+      birthDate: '01-01-2024',
+    };
+
+    const response = await axios({
+      url: 'http://localhost:3000',
+      method: 'post',
+      data: {
+        query: `
+            mutation Mutation($data: UserInput) {
+              createUser(data: $data) {
+                id
+                name
+                email
+                birthDate
+              }
+            }
+          `,
+        variables: {
+          data: user,
+        },
+      },
+    });
+
+    chai.expect(user.name).to.be.equal(response.data.data.createUser.name);
+    chai.expect(user.email).to.be.equal(response.data.data.createUser.email);
+    chai.expect(user.birthDate).to.be.equal(response.data.data.createUser.birthDate);
+
+    const createdUser = await dataSource.getRepository(User).findOneBy({
+      id: response.data.data.createUser.id,
+    });
+
+    if (createdUser) {
```

Comment:
não deveria ter esse if, o teste precisa sempre checar que o usuário foi criado
do jeito que ta, se createdUser for vazio o teste não vai dar erro (sendo que era pra dar erro)